itest/rust/src/property_test.rs

```diff
@@ -293,3 +294,78 @@ struct CheckAllExports {
 
 #[godot_api]
 impl CheckAllExports {}
+
+#[repr(i64)]
+#[derive(Property, Debug, PartialEq, Eq, Export)]
+pub enum TestEnum {
+    A = 0,
+    B = 1,
+    C = 2,
+}
+
+#[derive(GodotClass)]
+pub struct DeriveProperty {
+    #[var]
+    pub foo: TestEnum,
+}
+
+#[godot_api]
+impl DeriveProperty {}
+
+#[itest]
+fn derive_property() {
+    let mut class = DeriveProperty { foo: TestEnum::B };
+    assert_eq!(class.get_foo(), TestEnum::B as i64);
+    class.set_foo(TestEnum::C as i64);
+    assert_eq!(class.foo, TestEnum::C);
+}
+
+#[derive(GodotClass)]
+#[class(base = Object)]
```

Comment:
You can omit this line. Then the base is implicitly `RefCounted`, which means the object is destroyed at the end of scope. This should fix the memory leak in CI.

Below, you would then need to use `RefCountedVirtual`.